schema/schema.json

```diff
@@ -0,0 +1,122 @@
+{
+    "description": "Schema for OpenContainer bundle configuration file",
```

Comment:
i'm not keen on using `config.json`, even though this file is in a `./schema/` directory. At best, perhaps `config-schema.json`